docs/commands/join.md

```diff
@@ -1,59 +1,70 @@
+To easily distinguish the origin of OpenAPI objects and properties, you can optionally instruct the `join` command to append custom prefixes to them.
 
-Apart from providing individual API definition files as the input, you can specify the path to a folder that contains multiple API definition files and match them with a wildcard (for example, `myproject/openapi/*.yaml`). The `join` command collects all matching files and combines them into one file.
+The `join` command accepts both YAML and JSON files, which you can mix in the resulting `openapi.yaml` file. Setting a custom name for this file is not supported, which also means that if there is a file called `openapi.yaml` in the working directory, it will be overwritten.
 
-Only OpenAPI 3.x definitions are supported by this command.
+Apart from providing individual API definition files as the input, you can also specify the path to a folder that contains multiple API definition files and match them with a wildcard (for example, `myproject/openapi/*.yaml`). The `join` command will collect all matching files and combine them into one file.
 
+### Usage
 
-### `join` usage
-
+```bash
+openapi join <entrypoint> <entrypoint>...
+openapi join <path-to-folder>/<wildcard-pattern> [--lint]
+openapi join [--help] [--prefix-components-with-info-prop] [--prefix-tags-with-info-prop] [--prefix-tags-with-filename]
 
-```
-Positionals:
-entrypoints                                              [array] [default: []]
-
-Options:
---version                           Show version number.             [boolean]
---help                              Show help.                       [boolean]
---lint                              Lint definitions[boolean] [default: false]
---prefix-tags-with-info-prop        Prefix tags with property value from info
-object                            [string]
---prefix-tags-with-filename         Prefix tags with property value from file
-name            [boolean] [default: false]
---prefix-components-with-info-prop  Prefix components with property value from
-info object                       [string]
+openapi join first-api.yaml second-api.yaml
+openapi join first-api.yaml second-api.json
+openapi join ./*.yaml
+openapi join --version
 ```
 
+## Options
 
-The command:
+Option                               | Type               | Required     | Default     | Description
+-------------------------------------|:------------------:|:------------:|:-----------:|------------
+`entrypoints`                        | `array`, wildcard  | yes          | `[]`, -     | 1. Array of paths to API definition files that you want to join. At least two input files are required.<br />2. A wildcard pattern to match API definition files within a specific folder
```

Comment:
Question about the Default column - does this `[], - ` mean that there's no default value?

Reply:
Yes. It's better to leave a hyphen here

Reply:
Got it, thanks!